frontend/src/pages/Notifications/NotificationListItems.js

```diff
@@ -41,6 +42,12 @@ const styles = theme => ({
   }
 });
 
+const styling = {
```

Comment:
rename to `styles`

Reply:
moved to styles and changed color in palette